.kitchen.yml

```diff
@@ -3,13 +3,22 @@ driver:
   name: vagrant
 
 provisioner:
-  name: chef_solo
+  name: chef_zero
 
 platforms:
   - name: ubuntu-12.04
+  - name: ubuntu-14.04
   - name: centos-6.4
+  - name: centos-7.1
 
 suites:
   - name: default
     run_list:
+      - zabbix_ng::default
+      #- zabbix_ng::sender
```

Comment:
I think it's better to either mark commented out blocks like this with a `TODO` or simply remove them altogether. Similar like this:

```ruby
# TODO: zabbix_ng::sender
```